dashboard/app/views/levels/editors/fields/_blockly.html.haml

```diff
@@ -43,6 +43,7 @@
       %p Number is a multiplier for how long each step takes (so higher numbers are slower). Default is 5 for Maze, 2 for Bee.
       = f.number_field :step_speed
 
+  - if @level.is_a? GamelabJr
```

Comment:
again, seems like the best solution currently available even though it introduces conditionals.